tsk/base/tsk_base_i.c

```diff
@@ -16,7 +16,7 @@
 size_t
 strnlen(const char* s, size_t maxlen) {
   const char* const z = memchr(s, 0, maxlen);
-  return z ? z-s : n;
+  return z ? z-s : maxlen;
 }
 
 #endif
```

Comment:
Presumably post releases have a strlen?

Reply:
Everything I've ever compiled locally must, or I wouldn't have been able to compile.